src/shared/logger/activation.ts

```diff
@@ -159,4 +159,30 @@ async function registerLoggerCommands(context: vscode.ExtensionContext): Promise
             recordVscodeViewLogs()
         })
     )
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.viewLogsAtMessage', async (logID: number) => {
+            const msg: string | undefined = getLogger().getTrackedLog(logID, LOG_PATH)
+            await vscode.commands.executeCommand('aws.viewLogs')
```

Comment:
Just a heads-up, only logs written to the AWS Toolkit Logs channel is written to the file opened by this. This currently will not scale to logs written to any of the other output channels since they are not backed by files and are instead held ephemerally by VS Code.

Reply:
I thought the only logs relevant to users were written to file provided by `LOG_PATH`, but if there are more logs then the command will need to be modified to support different log channels.

Reply:
That's mostly true (we're admittedly not the most consistent in terms of where we log things to), but it runs a bit counter to the fact that you're building the functionality directly into the Winston logger impl and explicitly adding support for `channel://`. Output channels themselves have a button to clear all output, I don't think support operations to goto lines, and are not backed by files (technically the AWS Toolkit Logs channel isn't either, the file is a mirror that we write to separately).

That being said, I'm just pointing this out as a heads-up for two reasons: if we do add another separate logfile (most likely case would be a different feature creating its own logs), you'll have to modify this to accept a logfile as well, and don't expect this to scale to other log actions that would point to the output channels (e.g. SAM invocation failures)